config/initializers/load_pre_production_formats.rb

```diff
@@ -0,0 +1,13 @@
+def pre_production
+  pre_production_formats = []
+  Dir["lib/documents/schemas/*.json"].each do |file|
+    read_file = File.read(file)
+    parsed_file = JSON.parse(read_file)
+    if read_file.include? "\"pre_production\": true"
```

Comment:
Good idea, need to check it has the key and if so that the key contains "pre-production: true"